sklearn/linear_model/bayes.py

```diff
@@ -434,3 +477,42 @@ def fit(self, X, y):
         self.lambda_ = lambda_
         self._set_intercept(X_offset, y_offset, X_scale)
         return self
+
+    def predict(self, X, return_std=False):
+        """Predict using the linear model. In addition to the mean of the
+        predictive distribution, also its standard deviation can be returned.
+
+        See: http://www.utstat.toronto.edu/~rsalakhu/sta4273/notes/Lecture2.pdf
```

Comment:
It would be great to have this and a more formal reference in the class docstring, formatted as we usually do for references.